File: app/src/main/java/net/pside/android/example/mediaplayer/util/ExoPlayerUtil.java
package net.pside.android.example.mediaplayer.util;

import androidx.annotation.NonNull;

import com.google.android.exoplayer2.ExoPlayer;

public class ExoPlayerUtil {

    private ExoPlayerUtil() {
    }

    @NonNull
    public static String getPlaybackStateString(int playbackState) {
        String state;
        switch (playbackState) {
            case ExoPlayer.STATE_BUFFERING:
                state = "BUFFERING";
                break;
            case ExoPlayer.STATE_ENDED:
                state = "ENDED";
                break;
            case ExoPlayer.STATE_IDLE:
                state = "IDLE";
                break;
            case ExoPlayer.STATE_READY:
                state = "READY";
                break;
            default:
                state = "UNKNOWN";
        }
        return state;
    }

}

File: app/src/main/java/net/pside/android/example/mediaplayer/util/ExoPlayerUtil.kt
package net.pside.android.example.mediaplayer.util

import com.google.android.exoplayer2.ExoPlayer

object ExoPlayerUtil {
    fun getPlaybackStateString(playbackState: Int): String = when (playbackState) {
        ExoPlayer.STATE_BUFFERING -> "BUFFERING"
        ExoPlayer.STATE_ENDED -> "ENDED"
        ExoPlayer.STATE_IDLE -> "IDLE"
        ExoPlayer.STATE_READY -> "READY"
        else -> "UNKNOWN"
    }
}
